lib/std/fmt.zig

```diff
@@ -587,6 +594,18 @@ pub fn formatAsciiChar(
     return format(context, Errors, output, "\\x{x:0<2}", .{c});
 }
 
+pub fn formatUtf8Codepoint(
+    c: u21,
+    options: FormatOptions,
+    context: var,
+    comptime Errors: type,
+    output: fn (@TypeOf(context), []const u8) Errors!void,
+) Errors!void {
+    var buf: [4]u8 = undefined;
+    const len = std.unicode.utf8Encode(c, buf[0..]) catch unreachable;
```

Comment:
why is this catch unreachable? what if someone e.g. passed in `0x1fffff`

Reply:
> attempt to unwrap error: CodepointTooLarge

Personally I'd prefer to have `utf8EncodeUnsafe` with 0xFFFD replacement.